tests/flakyfunctional/xtriggers/01-workflow_state/flow.cylc

```diff
@@ -8,7 +8,7 @@
     initial cycle point = 2011
     final cycle point = 2016
     [[xtriggers]]
-        upstream = workflow_state("{{UPSTREAM}}//%(point)s/foo:data_ready"):PT1S
+        upstream = workflow_state("{{UPSTREAM}}//%(point)s/foo:data_ready", is_trigger=True):PT1S
```

Comment:
Behaviour change.

Turned off the implicit, "this isn't a status so handle it as a trigger" behaviour for xtriggers to make them explicit for safety.

Reply:
Why do this for the xtrigger but not the CLI?

Reply:
The same reason this feature was originally implemented, back-compat.

But we now seem to be ok with dropping this (which is much nicer).